dscecho/src/args.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use clap::Parser;
+
+#[derive(Debug, Parser)]
+#[clap(name = "dscecho", version = "1.0.0", about = "Debugging helper resource that echos the input.", long_about = None)]
+pub struct Args {
+    #[clap(short, long, help = "The input to the echo command as JSON.  If no input is provided the JSON schema for the input is returned.")]
```

Comment:
For a generic 'echo functionality' I don't think we should limit the input to be a JSON. It probably should be any string.

Reply:
What is echo'd doesn't have to be JSON, but the input has to be a valid JSON object that matches the resource schema